CHANGELOG/CHANGELOG-1.3.md

```
@@ -0,0 +1,964 @@
+* Resources
+  * Improved node stability by *optionally* evicting pods upon memory pressure - [Design Doc](https://github.com/kubernetes/kubernetes/blob/release-1.3/docs/proposals/kubelet-eviction.md)
+  * **alpha**: NVIDIA GPU support ([#24836](https://github.com/kubernetes/kubernetes/pull/24836), [@therc](https://github.com/therc))
+  * Adding loadBalancer services and nodeports services to quota system
+
+## Known Issues and Important Steps before Upgrading
+
+The following versions of Docker Engine are supported - *[v1.10](https://github.com/kubernetes/kubernetes/issues/19720)*, *[v1.11](https://github.com/kubernetes/kubernetes/issues/23397)*
+Although *v1.9* is still compatible, we recommend upgrading to one of the supported versions.
+All prior versions of docker will not be supported.
+
+#### ThirdPartyResource
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)